website/source/docs/internals/telemetry.html.md

```diff
@@ -50,742 +50,1468 @@ The following is an example telemetry dump snippet:
 [2017-12-19 20:38:00 +0000 UTC][S] 'vault.merkle.flushDirty': Count: 100 Min: 0.006 Mean: 0.024 Max: 0.098 Stddev: 0.019 Sum: 2.386 LastUpdated: 2017-12-19 20:38:09.848158309 +0000 UTC m=+10473.691527099
 ```
 
-You'll note that log entries are prefixed with the metric type as follows:
```

Comment:
Why remove this?

Reply:
Because we have instead included the metric type as a column in the table

Reply:
But actually it's about the preceding telemetry log dump, e.g.

> [2017-12-19 20:38:00 +0000 UTC]**[S]** 'vault.merkle.flushDirty': Count: 100 Min: 0.006 Mean: 0.024 Max: 0.098 Stddev: 0.019 Sum: 2.386 LastUpdated: 2017-12-19 20:38:09.848158309 +0000 UTC m=+10473.691527099